packages/manager/modules/hub/src/products/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,32 @@
+{
+  "manager_hub_products_CDN_DEDICATED": "CDN",
```

Comment:
Is there anyway to mutualize these translations instead of having them in triplicate ?

Reply:
A shared module for things that impact services ? (billing, common translations, paths, universe, catalog ? )

Reply:
Could be a module which would be the start of Entities:
```js
{
    "cdn": {
        displayName: "CDN"
    },
    "exchange": {
        displayName: "Server Exchange"
    },
}
```

something like that

Reply:
We need to give some thought I think, so I'll leave as is if that is ok for you but I keep that idea in mind